netbox_agent/logging.py

```diff
@@ -0,0 +1,23 @@
+import logging
+
+
+def init_log():
+    logger = logging.getLogger('netbox_agent')
+    logger.setLevel(logging.DEBUG)
```

Comment:
This should be configurable with the CLI arguments

Reply:
implemented!